File: Code/apps/Wroker_Server/problems/find-max-element/drivers/javascript/driver.js
// --- DRIVER CODE START ---
const fs = require('fs');

const DELIMITER = '$$$DELIMITER$$$';

try {
    if (!fs.existsSync('/app/input.txt')) throw new Error('Input file not found');
    const inputData = fs.readFileSync('/app/input.txt', 'utf-8');
    
    // Split by delimiter and filter empty lines
    const testCases = inputData.split(DELIMITER).filter(tc => tc.trim() !== '');

    testCases.forEach((testCase) => {
        // Split input by newlines to separate arguments
        const lines = testCase.trim().split('\n');
        
        // Arg 0: Number
        const arg0 = Number(lines[0]);
        // Arg 1: Smart Parse (JSON or Space-Separated)
        let arg1;
        const raw1 = lines[1].trim();
        if (raw1.startsWith('[')) {
            // It looks like JSON array
            arg1 = JSON.parse(raw1);
        } else {
            // Fallback: Space-separated (e.g. "1 2 3 4")
            arg1 = raw1.split(/\s+/).map(Number);
        }

        // Call User Function
        if (typeof findMax !== 'function') {
            throw new Error("Function 'findMax' not found. Did you name it correctly?");
        }
        
        const result = findMax(arg0, arg1);

        // Print Result
        console.log(result);
        console.log(DELIMITER);
    });

} catch (err) {
    // Print to Stderr so our Node.js Worker can catch it
    console.error("Driver Error:", err.message);
    process.exit(1);
}
// --- DRIVER CODE END ---

File: Code/apps/problems_directory/problems/find-maximum/drivers/python/driver.py
# --- DRIVER CODE START ---
import sys
import os
import json

DELIMITER = "$$$DELIMITER$$$"

def main():
    try:
        with open("/app/input.txt", "r") as f:
            input_data = f.read()
            
        test_cases = input_data.split(DELIMITER)
        sol = Solution()
        
        for test_case in test_cases:
            if not test_case.strip():
                continue
                
            lines = test_case.strip().split('\n')
            
            # Arg 0: Smart Parse
            raw_val = lines[0].strip()
            if raw_val.startswith("["):
                arg0 = json.loads(raw_val)
            else:
                arg0 = [int(x) for x in raw_val.split()]

            result = sol.findMax(arg0)
            print(result)
            print(DELIMITER)
            
    except Exception as e:
        sys.stderr.write(f"Driver Error: {str(e)}\n")
        sys.exit(1)

if __name__ == "__main__":
    main()
# --- DRIVER CODE END ---